src/core/lib/surface/init.cc

```diff
@@ -109,6 +109,7 @@ void grpc_init(void) {
       g_shutting_down_cv->SignalAll();
     }
     grpc_iomgr_init();
+    // Note: use AresInit() when we remove this function.
     grpc_resolver_dns_ares_init();
```

Comment:
To use `AresInit`, I think you can have something like:

```
if (IsEventEngineDnsEnabled()) {
  AresInit();
} else {
  grpc_resolver_dns_ares_init();
}
```

The same applies to shutdown.

Reply:
done, thanks!